packages/dev/core/src/Materials/Node/Blocks/Fragment/perturbNormalBlock.ts

```diff
@@ -279,6 +303,11 @@ export class PerturbNormalBlock extends NodeMaterialBlock {
         state.compilationString += this._declareOutput(this.output, state) + " = vec4(0.);\r\n";
         state.compilationString += state._emitCodeFromInclude("bumpFragment", comments, {
             replaceStrings: [
+                { search: /texture2D\(bumpSampler,vBumpUV\)/g, replace: `${uvForPerturbNormal}` },

```

Comment:
Was that a bug in the existing code?

Reply:
nope, this block is only in the case of object space :-) the other one has a + uvOffset